curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -66,12 +66,12 @@ async (getUserInput) => {
 };
 ```
 
-You can `POST` to `/api/exercise/add` with form data `userId=_id`, `description`, `duration`, and optionally `date`. If no date is supplied, the current date will be used. The response returned will be the user object with the exercise fields added.
+You can `POST` to `/api/users/{_id}/exercises` with form data `description`, `duration`, and optionally `date`. If no date is supplied, the current date will be used. The response returned will be the user object with the exercise fields added.
```

Comment:
```suggestion
You can `POST` to `/api/users/:_id/exercises` with form data `description`, `duration`, and optionally `date`. If no date is supplied, the current date will be used. The response returned will be the user object with the exercise fields added.
```

Should we use the format we used for the params in the URL shortener PR?

Reply:
Not a small nitpick at all! Thanks for catching that @nhcarrigan. I agree that we should be consistent with this.

I'll commit your suggestions now.